### PR TITLE
[octavia] bump utils to 0.16.1, fix migration job

### PR DESCRIPTION
**Pull request by bashar-alkhateeb**
- bump up to use utils v0.16.1
- force creating proxysql secret before starting the migration job
- add an or condition to configure proxysql in the job if any values are set in vaules in proxysql map, that to fix the first run issue caused by the lookup function on K8S secret which is not created on K8S side while helm generating the templates

### CODE REVIEW

File: openstack/octavia/values.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ db_name: octavia
 
 proxysql:
   mode: ""
+  forceSecretCreation: true
```
Comment (fwiesel) [inline openstack/octavia/values.yaml:30]: This can be dropped as well, no?

Comment (bashar-alkhateeb) [reply openstack/octavia/values.yaml:30]: i think it is better to be kept for the future, if anything changed in the secret, the secret will get updated before the job gets triggered , or is there a reason  to trigger the job before updating the proxysql config ?

Comment (bashar-alkhateeb) [reply openstack/octavia/values.yaml:30]: the job has already a weight of -1 which force helm to change the normal upgrade order

Comment (bashar-alkhateeb) [reply openstack/octavia/values.yaml:30]: well in the case of renaming even  `"helm.sh/hook-delete-policy": before-hook-creation` cannot remove the resource 
so should i remove the hook and we live will the risk that the job will be triggered before the secret got updated ? or keep it and naming changes will not remove the old secret ?

Comment (fwiesel) [reply openstack/octavia/values.yaml:30]: I'd remove the hook in the secret. If a job is relying on the secret to be there, then it fails right away and we can fix it accordingly.

**Pull request merged**